omero/sysadmins/whatsnew.rst

```diff
@@ -20,5 +20,7 @@ What's new for OMERO 5.4 for sysadmins
 - It is now possible to include an html template in the base `core_html` template so it can
   be used for all webclient pages.
 
+- The OMERO.web part of the :program:`omero admin diagnostics` command has been moved to :program:`omero web diagnostics`
```

Comment:
Do we want to emphasize on the fact that it is to avoid wrong report since omero.web can be installed separately from omero.server?

Reply:
Would make sense, yes.